news/2024/2024-02-21-monthly-beatmapping-contest-february-2024.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: post
+title: "Monthly Beatmapping Contest: February 2024"
+date: 2024-02-21 19:00:00 +0000
+---
+
+You already know you're the world's #1 mapper, but this contest can make everyone else realize it too.
+
+![](/wiki/shared/news/banners/monthly-beatmapping-contest.png)
+
+The [Monthly Beatmapping Contest](/wiki/Contests/Monthly_Beatmapping_Contest) (*MBC*) is a contest that runs monthly... most of the time. Our judges *may* have procrastinated a bit, but after hundreds of Discord highlights, the results are finally in!
+
+This month is all about **osu!** and **osu!mania**. Skip to whichever section you're interested in:
+
+- Results for [the previous osu!+osu!mania MBC](https://osu.ppy.sh/home/news/2023-11-14-monthly-beatmapping-contest-november-2023)
+  - [osu! results](#osu!-results)
+  - [osu!mania results](#osu!mania-results)
+- [Criteria for this month's MBC](#game-modes)
+
+## Previous contest results
+
+### osu! results
+
+This contest's **24** submissions were narrowed down to the **top 8** by our [screening crew](/wiki/Contests/Monthly_Beatmapping_Contest#screening). Those 8 maps were thoroughly evaluated by a team of judges, and unexpectedly, a mapper without any Ranked maps came out on top!
+
+[See the full results here.](https://mappersguild.com/contests/results?contest=65592eafaa5802f54edbe8c7)
+
+#### Summary
+
+| Placement | User | Beatmap | Badge prize | [Contest points](/wiki/Contests/Contest_points) |
+| :-- | :-- | :-- | :-- | :-- |
+| 1st + Screener's pick | [bIG data](https://osu.ppy.sh/users/17744610) | [yaseta - Dreamy Goat](https://osu.ppy.sh/beatmapsets/2135463) | ![](https://assets.ppy.sh/profile-badges/mbc-2020-1.png) | +3 (3 total) |
+| 2nd | [fowwo](https://osu.ppy.sh/users/4547551) | [siqlo - Me & U](https://osu.ppy.sh/beatmapsets/2135872) | - | +1 (1 total) |
+| 3rd | [Sing](https://osu.ppy.sh/users/3795679) | [ALEPH - IMPULSE](https://osu.ppy.sh/beatmapsets/2135262) | - | - |
+
+#### Top 10
+
+<div align="center">
```

Comment:
this adds margin after embeds to match regular paragraphs - looks better, i suggest using this everywhere

fa posts look fine since they have embeds above headings, which have top margins

```suggestion
<div align="center" class="osu-md__paragraph">
```